php/README.md

```diff
@@ -48,26 +48,31 @@ Search an object by query
 php ./bin/cli.php objects:get [objectType] --query=test
 ```
 
-Creare new object
+Create new object
 
 ```bash
-./bin/cli.php objects:create [objectType] name=newCompany city=Cambridge state=Massachusetts
+./bin/cli.php objects:create [objectType] [properties]
 ```
 
-Please also notice that some objects require mandatory properties, that you can provide in the following way:
-
-Delete an object by Id
-
+For example:
 ```bash
-php ./bin/cli.php objects:delete [objectType] [objectId]
+./bin/cli.php objects:create companies name=newCompany city=Cambridge state=Massachusetts
 ```
 
+Please also notice that some objects require mandatory properties.
+
 Update an object by Id
 
 ```bash
 php ./bin/cli.php objects:update [objectType] [objectId] [properties]
 ```
 For example:
 ```bash
-php ./bin/cli.php objects:update contacts 123456 firstname=Josh
+php ./bin/cli.php objects:update contacts 123456 firstname=Ryan
```

Comment:
To be consistent, lets either use `php` command or not use. At the moment some examples start from `php`, some - don't